uvloop/__init__.pyi

```diff
@@ -5,10 +5,25 @@ from collections.abc import Callable, Coroutine
 from contextvars import Context
 from typing import Any, TypeVar
 
-
 _T = TypeVar('_T')
 
 
+class Loop(AbstractEventLoop):
+    ...
+
+
+def new_event_loop() -> Loop:
```

Comment:
this is needed for `async with asyncio.Runner(loop_factory=uvloop.new_event_loop)`